src/container/__tests__/RelayContainer_hasFragmentData-test.js

```diff
@@ -34,13 +34,14 @@ describe('RelayContainer', () => {
           foo: () => Relay.QL`fragment on Node{id}`,
         },
       });
+      const relayContext = new RelayContext();
       const RelayTestRenderer = RelayTestUtils.createRenderer();
       mockContainerInstance = RelayTestRenderer.render(genMockPointer => {
         return <MockContainer foo={genMockPointer('42')} />;
-      });
+      }, relayContext);
```

Comment:
supernit: we generally prefer each argument on its own line unless it's a single argument - it makes it easy to miss the last argument otherwise.